apps/cnquery/cmd/builder/parse.go

```diff
@@ -141,6 +141,9 @@ func ParseTargetAsset(cmd *cobra.Command, args []string, providerType providers.
 		if err != nil {
 			log.Error().Err(err).Msg("cannot parse target")
 		}
+		if identityFile == "" && password == "" {
```

Comment:
I think this does not catch all valid cases. There is also aws instance connect and ssm

Reply:
You don't have to provide creds for ic/ssm (far as I know), aws generates a temporary keypair there

Reply:
You are right, this is covered as a separate asset type. never mind.